helm-charts/0.22.4/charts/sentinel/templates/Deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - name: spire-agent-socket
               mountPath: /spire-agent-socket
               readOnly: true
-          {{- if .Values.global.sentinelInitCommand.enabled }}
+          {{- if .Values.initCommand.enabled }}
```

Comment:
@abhishek44sharma I’m reverting this. — TCx figured out a way to use this value w/o having to resort to the `global` namespace. — FYI.